gb-backend-e2e/build.gradle

```diff
@@ -0,0 +1,43 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        maven { url 'https://repo.grails.org/grails/core' }
+    }
+}
+
+apply plugin: 'groovy'
+apply plugin: 'java'
+
+ext {
+    jacksonVersion = '2.9.5'
```

Comment:
Not 2.9.8? (https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-core)

Reply:
updated!